src/db.ts

```diff
@@ -149,10 +149,10 @@ export class Db implements OperationParent {
    * Creates a new Db instance
    *
    * @param databaseName - The name of the database this instance represents.
-   * @param topology - The server topology for the database.
+   * @param client - The MongoClient for the database.
    * @param options - Optional settings for Db construction
    */
-  constructor(databaseName: string, topology: Topology, options?: DbOptions) {
+  constructor(databaseName: string, client: MongoClient, options?: DbOptions) {
```

Comment:
```suggestion
  constructor(client: MongoClient, databaseName: string, options?: DbOptions) {
```
nit, but if we're changing this anyway, let's try to go from most fundamental element to most optional element in the parameter ordering (this mirrors how we store things in our private `s` object)